src/conf.py

```diff
@@ -125,6 +125,7 @@ def json_load(path):
 
 # NOTE: do not move to /tmp
```

Comment:
@lsh-0 left written not to move these caches to `/tmp`, but we definitely need a more stable location as the project folder will be often empty during builds (it's a new Jenkins workspace). Only on `lax--*` servers these will work well, but in the `elife-libraries--*` one that builds PR and the `develop` branch the cache will always be empty. We could have a configuration in the .cfg file to choose the root directory for caches?